traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -220,7 +220,7 @@ func (job *InvalidationJob) Read() ([]interface{}, error, error, int) {
 	if err != nil {
 		return nil, nil, fmt.Errorf("getting accessible tenants for user - %v", err), http.StatusInternalServerError
 	}
-	if len(where) > 0 {
+	if len(where) == 0 {
```

Comment:
This shouldn't be reversed

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.